### ISSUE TITLE
Only displays when editing 'color' attribute.

### ISSUE THREAD

**Issue by ktersius** [leehooi/vs-color-picker#8]
[
![screen-to-gif- picker](https://cloud.githubusercontent.com/assets/6417718/24189283/0266457c-0eed-11e7-8216-56928a56a965.gif)
](url)

**Comment by knight1O1** [comment]
1. It seems like the bug lies with "Hex color code",not with attribute of elements.
2. When u give the color using "rgb();" or write the color name directly,then it doesn't detects.But when u type the "#" for Hex color code,then it invokes.

![color](https://user-images.githubusercontent.com/29153455/27506178-b90298ee-58d0-11e7-912a-fe961e0a8c98.png)

**Comment by leehooi** [comment]
@knight1O1  Yes, you are right. It only be triggered when type '#' after ':'. 
The attribute has lot of possibilities: background, background-color, border, border-left, border-color ....
It is hard to enumerate all css attributes.
I'm finding a smart approach.
